src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java

```diff
@@ -491,15 +496,156 @@ private static boolean getPredicateForResponseTypeSpec(CaseData caseData, Respon
+                    && (RespondentResponseTypeSpec.FULL_DEFENCE
+                    .equals(caseData.getRespondent1ClaimResponseTypeForSpec())
+                    || RespondentResponseTypeSpec.FULL_DEFENCE
+                    .equals(caseData.getRespondent2ClaimResponseTypeForSpec()));
+            case ONE_V_TWO_TWO_LEGAL_REP:
+                //scenario: latest response is full defence
+                return caseData.getRespondent1ClaimResponseTypeForSpec() != null
+                    && caseData.getRespondent2ClaimResponseTypeForSpec() != null
+                    && caseData.getRespondent1ResponseDate() != null
+                    && caseData.getRespondent2ResponseDate() != null
+                    && !caseData.getRespondent1ClaimResponseTypeForSpec()
+                    .equals(caseData.getRespondent2ClaimResponseTypeForSpec())
+                    && ((RespondentResponseTypeSpec.FULL_DEFENCE
+                    .equals(caseData.getRespondent2ClaimResponseTypeForSpec())
+                    && caseData.getRespondent2ResponseDate().isAfter(caseData.getRespondent1ResponseDate()))
+                    || (RespondentResponseTypeSpec.FULL_DEFENCE
+                    .equals(caseData.getRespondent1ClaimResponseTypeForSpec())
+                    && caseData.getRespondent1ResponseDate().isAfter(caseData.getRespondent2ResponseDate())));
+            case TWO_V_ONE:
+                return (RespondentResponseTypeSpec.FULL_DEFENCE.equals(caseData.getClaimant2ClaimResponseTypeForSpec())
+                    || RespondentResponseTypeSpec.FULL_DEFENCE.equals(caseData.getClaimant1ClaimResponseTypeForSpec()))
+                    && !(RespondentResponseTypeSpec.FULL_DEFENCE.equals(caseData.getClaimant2ClaimResponseTypeForSpec())
+                    && RespondentResponseTypeSpec.FULL_DEFENCE.equals(caseData.getClaimant1ClaimResponseTypeForSpec()));
+            default:
+                return false;
+        }
+    }
+
+    public static final Predicate<CaseData> divergentRespondGoOfflineSpec = caseData ->
+        isDivergentResponsesGoOfflineSpec(caseData);
+
+    private static boolean isDivergentResponsesGoOfflineSpec(CaseData caseData) {
+
+        if (!SPEC_CLAIM.equals(caseData.getSuperClaimType())) {
+            return false;
+        }
+
+        switch (getMultiPartyScenario(caseData)) {
+            case ONE_V_TWO_TWO_LEGAL_REP:
+                return caseData.getRespondent1ClaimResponseTypeForSpec() != null
+                    && caseData.getRespondent2ClaimResponseTypeForSpec() != null
+                    && caseData.getRespondent1ResponseDate() != null
+                    && caseData.getRespondent2ResponseDate() != null
+                    && !caseData.getRespondent1ClaimResponseTypeForSpec()
+                    .equals(caseData.getRespondent2ClaimResponseTypeForSpec())
+                    //scenario: latest response is not full defence
+                    && (((!RespondentResponseTypeSpec.FULL_DEFENCE
```

Comment:
Could use the isAfter?A:B of the previous comment. Maybe it's even simpler to use small methods with meaningful names instead of this many boolean operators and comments

Reply:
Left to a less time-pressed occasion